src/Styles/Carousel.css

```diff
@@ -0,0 +1,46 @@
+.containerCarousel{
+    width: 600px;
```

Comment:
You can make use of @media https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries to make responsive design, notice that mobile view does not fit properly to the screen